ScarletEngine/src/Renderer/SceneProxy.cpp

```diff
@@ -0,0 +1,14 @@
+#include "Renderer/SceneProxy.h"
+
+namespace ScarletEngine
+{
+	void SceneProxy::DrawSMC(const Transform& Trans, const StaticMeshComponent& SMC)
```

Comment:
out of curiosity why is this called DrawSMC (and not just AddSMC)? Kind of confusing